### DISCUSSION TITLE
Why is there such a warning?

### DISCUSSION THREAD

**Discussion by zoushucai** [quarto-dev/quarto-cli#6778] [Q&A]
### Description

Hello, I am building a simple website using Quarto. I saw many warnings by opening the browser console. May I ask what happened to these warnings and can they be resolved?

<img width="987" alt="iShot_2023-09-09_22 26 03" src="https://github.com/quarto-dev/quarto-cli/assets/15843338/71ce81f4-8865-44d6-bf16-dbcd0b9efd7d">

Did I see here and say it was resolved? But actually it doesn't seem to have been resolved?
[https://github.com/quarto-dev/quarto-cli/discussions/5793](https://github.com/quarto-dev/quarto-cli/discussions/5793)

```
⚡ quarto --version
1.3.450
```

**Comment by zoushucai** [comment]
Okay, I thought my version was already the latest, but it still needs to be updated. I'll give it a try

**Comment by zoushucai** [comment]
However, these warnings were not successfully eliminated, although they had little impact

<img width="987" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/15843338/373115c6-c1a7-4f48-bd68-97dd4cba8d6a">

```
⚡ quarto --version
1.4.357
```

**Comment by mcanouil** [comment]
Please consider sharing a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by mcanouil** [reply]
PS: it would have been better to continue on the original thread rather than opening a duplicate of:
- #5793

**Reply by zoushucai** [reply]
Okay, thank you. I have moved on to the original question
